base/cvd/cuttlefish/host/commands/run_cvd/launch/BUILD.bazel

```diff
@@ -72,6 +72,16 @@ cf_cc_library(
     ],
 )
 
+cf_cc_library(
+    name = "common",
+    srcs = ["common.cpp"],
+    hdrs = ["common.h"],
```

Comment:
`common` isn't that descriptive of a name.

Can this be `enable_multitouch.{cc,h}` instead?

Reply:
Done